GeoFire.xcodeproj/project.pbxproj

```diff
@@ -22,19 +22,19 @@
 /* End PBXAggregateTarget section */
 
 /* Begin PBXBuildFile section */
+		38FC6CB01CFBE57C002AB722 /* GeoFireTest.m in Sources */ = {isa = PBXBuildFile; fileRef = FB8C8845196D55630061AC0F /* GeoFireTest.m */; };
```

Comment:
Generally, I'm not sure we need to check in the Pods products (unless we're using it for local dependency management?)